primitives/src/vid/advz.rs

```diff
@@ -140,17 +140,21 @@ where
     #[serde(with = "canonical")]
     poly_commits: Vec<P::Commitment>,
     all_evals_digest: V::NodeValue,
+    elems_len: usize,
 }
 
 // We take great pains to maintain abstraction by relying only on traits and not
 // concrete impls of those traits. Explanation of trait bounds:
 // 1,2: `Polynomial` is univariate: domain (`Point`) same field as range
 // (`Evaluation'). 3,4: `Commitment` is (convertible to/from) an elliptic curve
 // group in affine form. 5: `H` is a hasher
+//
+// `PrimeField` needed only because `bytes_to_field` needs it.
+// Otherwise we could relax to `FftField`.
 impl<P, T, H, V> VidScheme for GenericAdvz<P, T, H, V>
 where
     P: UnivariatePCS<Point = <P as PolynomialCommitmentScheme>::Evaluation>,
-    P::Evaluation: FftField,
```

Comment:
Any specific reason to bound it on `PrimeField`?

Reply:
I was in a rush. 😊 I'm happy to extend it to `Field` if you request it. That gives me an excuse to allocate the time to do it.

Reply:
The answer to your question is here: https://github.com/EspressoSystems/jellyfish/blob/474fcdf6a40102ce0acfc5c3729ed539562c5051/primitives/src/vid/advz.rs#L152-L153

This https://github.com/EspressoSystems/jellyfish/pull/381#discussion_r1349016039 is an explanation for why I restricted `bytes_to_field` to `PrimeField`.

Reply:
Got it. We can settle with `PrimeField` for now, and add a follow-up issue for the future improvement.

Reply:
https://github.com/EspressoSystems/jellyfish/issues/382